src/commands/eightball.ts

```diff
@@ -5,6 +5,7 @@ export const eightBall: CommandDef = {
   prefix: 'eightball',
   description:
     'Think of your question and get an answer from a magic pool ball!',
+  usage: `\`eightball [message]\``,
```

Comment:
This is incorrect, this command doesn't take a message. This might of been done first and just forgotten about.
Please change to:
```suggestion
  usage: 'eightball',
```